### PR TITLE
 custom connector in camunda

### PR DESCRIPTION
**Pull request by pradeepkodi15**
custom connector in camunda

### CODE REVIEW

File: content/user-guide/process-engine/connectors.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ connectors. Currently the following connector implementations exist:
   </tr>
 </table>
 
+It is also possible to implement your own custom connector in camunda. For more information about extending connectors please see the corresponding [section]({{< ref "/reference/connect/extending-connect/" >}}) of the [User Guide]({{< ref "/user-guide/_index.md" >}})
```
Comment (mboskamp) [inline content/user-guide/process-engine/connectors.md:32]:
```suggestion
It is also possible to implement your own custom connector in camunda. For more information about extending connectors please visit the [Connector reference]({{< ref "/reference/connect/extending-connect/" >}}). 
```

Comment (mboskamp) [reply content/user-guide/process-engine/connectors.md:32]: I would suggest to remove the lin to the user guide, as the important link is to the Connectors reference which is not related to a user guide.

**Comment by mboskamp** [comment]
Hi @pradeepkodi15, thanks for the addition. I have a small edit suggestion but other than that I will be happy to merge it. :)

**Comment by pradeepkodi15** [comment]
Thank you.. I adjusted your suggestion...

**Pull request merged**